gengokit/httptransport/templates/server.go

```diff
@@ -49,79 +46,6 @@ var ServerDecodeTemplate = `
 // to fail. So don't change them purely because you think they look a little
 // funny.
 
-// PathParamsTemplate is a source code literal of the code for the PathParams
```

Comment:
It is great to see this code removed, maybe @hasLeland feels sad about it because he wrote it, though he is probably glad that we don't have to maintain it anymore.